mlrun/datastore/base.py

```diff
@@ -64,6 +64,11 @@ def is_structured(self):
     def is_unstructured(self):
         return True
 
+    @staticmethod
+    def get_parsed_url(url):
```

Comment:
Please add a docstring to explain that we filter query params

Reply:
I'd rename it too. It's not really clear why `get_parsed_url` takes a URL in string form and returns a URL in string form. From the name and implied signature, it's impossible to guess what this function does.

Reply:
renamed